src/components/Input/TextField/TextFieldProps.ts

```diff
@@ -0,0 +1,17 @@
+import { ChangeEvent, FormEvent } from 'react';
+
+type SizeType = 'small' | 'medium' | 'large' | undefined;
+type StatusType = 'success' | 'error' | undefined;
```

Comment:
```suggestion
type SizeType = 'small' | 'medium' | 'large';
type StatusType = 'success' | 'error';
```

ini gaperlu dibikin atau undefined.. di interfacenya kasih tanda tanya aja